test/libsolidity/syntaxTests/controlFlow/uninitializedAccess/reverting_call_virtual2.sol

```diff
@@ -0,0 +1,19 @@
+abstract contract B
+{
+        function iWillRevert() pure public virtual { }
+
+        function test(bool _param) pure external returns(uint256)
+        {
+                if (_param) return 1;
+
+                iWillRevert();
+        }
+}
+
+contract C is B
+{
+        function iWillRevert() pure public override { revert(); }
+}
+
+// ----
+// Warning 6321: (135-142): Unnamed return variable can remain unassigned. Add an explicit return with value to all non-reverting code paths or name the variable.
```

Comment:
Can you add some additional info about the "current contract" in the analysis to the error message if this is not the contract of the function under analysis? Otherwise I can imagine this to be very confusing.

Reply:
It looks like either this test has not been updated yet or the code for inserting contract name does not work.

Reply:
Usually, the current contract is not mentioned if it is just the same as where the function is defined.. but maybe for abstract contracts we should always add it?